awx/main/scheduler/kubernetes.py

```diff
@@ -51,6 +53,81 @@ def list_active_jobs(self, instance_group):
 
         return pods
 
+    def create_secret(self, job):
+        registry_cred = job.execution_environment.credential
+        host = registry_cred.get_input('host')
+        scheme = 'https'
+        # urlparse requires '//' to be provided if scheme is not specified
+        original_parsed = urlparse.urlsplit(host)
+        if (not original_parsed.scheme and not host.startswith('//')) or original_parsed.hostname is None:
+            host = '%s://%s' % (scheme, host)
```

Comment:
could you remove `scheme` and just do

```python
host = 'https://%s' % host
```

It feels distracting to have a static variable definition, which then gets used once.

Reply:
@AlanCoding good point.  That will be cleaner.